front_end_pocket_poll/lib/imports/api_manager.dart

```diff
@@ -24,6 +25,12 @@ Future<ResultStatus<String>> makeApiRequest(
     };
 
     try {
+      //add the app version to the request payload
+      PackageInfo packageInfo = await Globals.getPackageInfo();
+      requestContent[RequestFields.PAYLOAD]
+          .putIfAbsent(appVersion, () => packageInfo.version);

```

Comment:
So i guess it really doesn't matter, but it might be better to use packageInfo.buildNumber. The version number is the arbitrary string we give the app (we're using semantic versioning so it will be like 1.0.0). Whereas the buildNumber is the unique sequential number that each app version must have.

Reply:
Will update to ensure uniqueness.